schema/artist/index.js

```diff
@@ -111,6 +111,23 @@ const ArtistType = new GraphQLObjectType({
       deathday: {
         type: GraphQLString,
       },
+      formatted_nationality_and_birthday: {
+        type: GraphQLString,
+        resolve: ({ birthday, nationality }) => {
+          let formatted_bday = (!isNaN(birthday) && birthday) ? 'b. ' + birthday : birthday;
+
+          if (formatted_bday && formatted_bday.toLowerCase().includes('born')) {
+            const regEx = new RegExp('born', 'ig');
+            formatted_bday = formatted_bday.replace(regEx, 'b.');
```

Comment:
JS has a regex literal and you also shouldn’t need to use the greedy (`g`) option as `born` will only be in there once, if any, no?

E.g.

```js
formatted_bday = formatted_bday.replace(/born/i, 'b.');
```